node/config/types.go

```diff
@@ -732,3 +734,25 @@ type IndexConfig struct {
 	// EnableMsgIndex enables indexing of messages on chain.
 	EnableMsgIndex bool
 }
+
+type SturdyDB struct {
+	// HOSTS is a list of hostnames to nodes running YugabyteDB
+	// in a cluster. Only 1 is required
+	Hosts []string
+
+	// The Yugabyte server's username with full credentials to operate on Lotus' Database. Blank for default.
+	Username string
+
+	// The password for the related username. Blank for default.
+	Password string
+
+	// The database (logical partition) within Yugabyte. Blank for default.
+	Database string
+
+	// The port to find Yugabyte. Blank for default.
+	Port string
+
+	// ITest is for optimized integration testing and not
+	// for production. Blank for default production configuration.
+	ITest string
```

Comment:
Shouldn't need to be here

Reply:
It's a schema picker so we can parallelize integration tests...
1. simply for single-binary tests.
2. for multi-binary tests using configuration.